Source/FDWaveformView.swift

```diff
@@ -78,6 +78,14 @@ open class FDWaveformView: UIView {
     /// Whether to allow tap and pan gestures to change highlighted range
     /// Pan gives priority to `doesAllowScroll` if this and that are both `true`
     /*@IBInspectable*/ open var doesAllowScrubbing = true
+    
+    /// Supported bounds for scrubbing
+    public enum ScrubbingBound {
+        case lower, upper
```

Comment:
I went for `lower` and `upper` as they match `highlightedSamples.lowerBound` and `highlightedSamples.upperBound`, but could equally be `start` and `end`.

Reply:
Also, the enum could just be called `Bound` if you'd prefer?